client/extensions/woocommerce/app/settings/payments/stripe/style.scss

```diff
@@ -93,3 +97,7 @@
 		@include placeholder();
 	}
 }
+
+.form-fieldset {
```

Comment:
This should probably be more specific 

<img width="245" alt="Screenshot 2019-08-08 at 22 32 41" src="https://user-images.githubusercontent.com/43215253/62739337-7bcef980-ba2c-11e9-960b-4e0a387759bf.png">

Reply:
It probably should, eh? Though I think this belongs in a separate issue.

Reply:
Sorry, to clarify, the image above is a regression which is caused on this branch by that selector not being specific enough. :)

Reply:
Thanks for catching that! :)